packages/dashboard/src/components/DoubleBarChart/index.js

```diff
@@ -0,0 +1,97 @@
+import React, { useContext } from 'react';
+import PropTypes from 'prop-types';
+import { ThemeContext } from 'styled-components';
+import { Chart, Dataset } from 'react-rainbow-components';
+import datalabels from 'chartjs-plugin-datalabels';
+import handleData from './helpers/handleData';
+import { Legend, LegendItem } from './legend';
+import { StyledContainer } from './styled';
+import getOptions from './helpers/getOptions';
+
+const plugins = [datalabels];
+const datalabelsConf = {
+    anchor: 'end',
+    align: 'top',
+    display: 'auto',
+};
+const defaultColors = {
+    rainbow: {
+        palette: {
+            brand: {
+                main: '#01B6F6',
+            },
+            border: {
+                divider: '#D7D9E2',
+            },
+        },
+    },
+};
+
+const DoubleBarChart = (props) => {
+    const { titles, data, xLabel, yLabel } = props;
+    const { rainbow } = useContext(ThemeContext) || defaultColors;
```

Comment:
there is a useTheme hook that does this for you `react-rainbow-components/libs/hooks/useTheme.js`